perun-core/src/test/java/cz/metacentrum/perun/core/impl/modules/attributes/urn_perun_user_attribute_def_virt_voPersonExternalAffiliationTest.java

```diff
@@ -118,6 +128,33 @@ public void setVariables() {
 		groupAtt1.setValue(arrListValue);
 	}
 
+	@Test
+	public void getAttributeValueNoDuplicities() throws Exception {
+		System.out.println("getAttributeValueNoDuplicities()");
+		urn_perun_user_attribute_def_virt_voPersonExternalAffiliation classInstance = new urn_perun_user_attribute_def_virt_voPersonExternalAffiliation();
+		PerunSessionImpl session = mock(PerunSessionImpl.class, RETURNS_DEEP_STUBS);
+
+		when(session.getPerunBl().getUsersManagerBl().getUserExtSources(session, user)).thenReturn(
+			Arrays.asList(ues1, ues2, ues3)
```

Comment:
It confused me a bit at first that we return three ext sources here and only get the values from one of them. Maybe it would more semantically correct if we only returned ues3 here (or mocked the attributes for the other ext sources)?